acme/acme/client.py

```diff
@@ -609,6 +609,10 @@ def _send_request(self, method, url, *args, **kwargs):
                       method, url, args, kwargs)
         kwargs['verify'] = self.verify_ssl
         kwargs.setdefault('headers', {})
+        """
+          PULL REQUEST: fix bug where newline creeps in.
+        """
+        self.user_agent = self.user_agent.replace( '\n', '' )
```

Comment:
PEP8 strict checking is enabled for the `acme` module, and that's causing [a lint test failure](https://travis-ci.org/certbot/certbot/jobs/135280656#L609) with these whitespaces.